specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/examples/PublicIpPrefixGet.json

```diff
@@ -0,0 +1,26 @@
+{
+    "parameters" : {
+      "api-version": "2018-07-01",
+      "subscriptionId" : "subid",
+      "resourceGroupName": "rg1",
+      "publicIpPrefixName": "test-ipprefix"
+    },
+    "responses" : {
+      "200" : {
+        "body" : {
+          "name" : "test-ipprefix",
+          "id" : "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/publicIPPrefixes/test-ipprefix",
+          "location" : "westus",
+          "properties" : {
+            "provisioningState" : "Succeeded",
+            "publicIPAddressVersion" : "IPv4",
+            "idleTimeoutInMinutes" : 4,
+            "ipTags" : [
```

Comment:
We're exposing tags as part of the prefix? 

Also, should it align with this: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2018-06-01/examples/PublicIpAddressList.json